common/taskana-common/src/main/java/pro/taskana/common/internal/configuration/TaskanaConfigurationInitializer.java

```diff
@@ -158,6 +163,89 @@ Optional<T> initialize(
         Properties properties, String separator, Field field, TaskanaProperty taskanaProperty);
   }
 
+  static class MapPropertyParser implements PropertyParser<Map<?, ?>> {
```

Comment:
comment somewhere that this is the idea behind the MapPropertyParser: `<Property>.<Key> = <value>`

Reply:
done